config/manifests/bases/cockroach-operator.clusterserviceversion.yaml

```diff
@@ -23,46 +23,94 @@ metadata:
     createdAt: CREATED_AT_PLACEHOLDER
     description: CockroachDB Operator
     operators.operatorframework.io/builder: operator-sdk-v1.1.0
-    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/cockroachdb/cockroach-operator
     support: Cockroach Labs
-  name: cockroach-operator.vX.Y.Z
+  name: cockroach-operator.v0.0.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
+    - description: CrdbCluster is the CRD for the cockroachDB clusters API
+      displayName: CoachroachDB Operator
+      kind: CrdbCluster
+      name: crdbclusters.crdb.cockroachlabs.com.cockroachlabs.com
+      resources:
+      - kind: Service
+        name: cockroach-operator
+        version: v1
+      - kind: StatefulSet
+        name: cockroach-operator
+        version: v1
+      specDescriptors:
+      - description: Number of nodes (pods) in the cluster
+        displayName: Number of nodes
+        path: nodes
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podCount
```

Comment:
We have multiple descriptors that need some TLC in the UI.  Does this work in the OpenShift UI?

Reply:
I haven't checked this out in the UI, but this is autogenerated and there aren't any new descriptor types here. Was there one (or more) in particular you want to validate?